ui/components/ui/form-field/form-field.js

```diff
@@ -163,6 +178,15 @@ export default function FormField({
             {passwordStrengthText}
           </Typography>
         )}
+        {customSpendingCapText && (
+          <Typography
+            color={COLORS.TEXT_DEFAULT}
+            variant={TYPOGRAPHY.H7}
+            className="form-field__custom-spending-cup-text"
+          >
+            {customSpendingCapText}
+          </Typography>
+        )}
```

Comment:
We are still adding feature specific content to the ui components? Can this be added in the spending cap component?

Reply:
I missed this in my review, definitely don't need to be passing props to let generic components be aware of their higher level consumer's contexts like this.

Reply:
Yes, I added this in the spending cap component. Sorry for late fixing @georgewrmarshall and @brad-decker, I was on vacation. Done [here](https://github.com/MetaMask/metamask-extension/pull/15522/commits/1de7f03ba5fa31864eeaf3d69bf7ae9be95d603f).

Reply:
Hope your vacation was restorative @VSaric , glad to have you back!